.github/workflows/ci.yml

```diff
@@ -27,33 +27,45 @@ jobs:
       - uses: Swatinem/rust-cache@v2
 
       # Install OpenSSL Libraries for Windows
-      - name: install openssl
+      # - name: install openssl
+      #   if: matrix.os == 'windows-latest'
+      #   run: |
+      #     $ErrorActionPreference = "Stop"
+      #     choco install --verbose openssl
+      #     openssl version
+      #     refreshenv
+      #     echo "OPENSSL_LIB_DIR=C:/Program Files/OpenSSL-Win64/lib" >> $env:GITHUB_ENV
+      #     echo "OPENSSL_DIR=C:/Program Files/OpenSSL-Win64/" >> $env:GITHUB_ENV
+      #     echo "OPENSSL_INCLUDE_DIR=C:/Program Files/OpenSSL-Win64/include" >> $env:GITHUB_ENV
+      # Only test Rustls on Windows instead due to #1191
+      - name: "Interim Hacky Windows Test for #1191"
         if: matrix.os == 'windows-latest'
         run: |
-          $ErrorActionPreference = "Stop"
-          choco install --verbose openssl
-          openssl version
-          refreshenv
-          echo "OPENSSL_LIB_DIR=C:/Program Files/OpenSSL-Win64/lib" >> $env:GITHUB_ENV
-          echo "OPENSSL_DIR=C:/Program Files/OpenSSL-Win64/" >> $env:GITHUB_ENV
-          echo "OPENSSL_INCLUDE_DIR=C:/Program Files/OpenSSL-Win64/include" >> $env:GITHUB_ENV
+          sed -i '0,/openssl/s//rustls/' kube/Cargo.toml
+          cat kube/Cargo.toml
```

Comment:
Is this for debugging? Should it be omitted? Or perhaps a `grep` to avoid printing the whole thing out?